schemas/_common-schema.json

```diff
@@ -167,6 +167,17 @@
           "const": 2,
           "description": "Version of the RUM event format",
           "readOnly": true
+        },
+        "session": {
+          "type": "object",
+          "description": "Session-related internal properties",
+          "properties": {
+            "plan": {
```

Comment:
what about putting plan field required in session object?

Reply:
but... didn't we say that we *cannot* add required fields ? :suspect:

Reply:
at the root yes, but in a new optional object, it should not be an issue, right?

Reply:
Since `session` is a new field (optional), I guess we are free to have `plan` as required?

Reply:
👌 didn't see it was under `_dd`